spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,199 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController do
+  describe 'POST #create' do
+    context 'When create a book suggestion' do
+      let!(:editorial) { Faker::Book.publisher }
+      let!(:price)     { Faker::Commerce.price }
+      let!(:author)    { Faker::Book.author }
+      let!(:title)     { Faker::Book.title }
+      let!(:link)      { Faker::Internet.url }
+      let!(:publisher) { Faker::Book.publisher }
+      let!(:year)      { Faker::Number.between(1950, Time.zone.today.year) }
```

Comment:
what about using `attributes_for`? https://stackoverflow.com/questions/13150272/meaning-for-attributes-for-in-factorygirl-and-rspec-testing
It could make this smaller and easier

Reply:
Reading, thanks! :)